src/chrome/content/rules/SONGLYRICS.com.xml

```diff
@@ -0,0 +1,24 @@
+<!--
+	Bad gateway:
+		- m.songlyrics.com
+
+	MCB:
+		- http://l.yimg.com/pb/webplayer/0.9.30/js/player-min.js
+
+	Server not found:
```

Comment:
There are no DNS records for them, so you can remove the comments.

Reply:
Done.

Reply:
Thanks!